plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchTableHandle.java

```diff
@@ -48,6 +52,7 @@ public ElasticsearchTableHandle(Type type, String schema, String index, Optional
         this.query = requireNonNull(query, "query is null");
 
         constraint = TupleDomain.all();
+        regexes = new HashMap<>();
```

Comment:
use immutable collection to keep the table handle class immutable

```
ImmutableMap.of()
```

Reply:
Done